client/signup/steps/plans/index.jsx

```diff
@@ -114,40 +115,76 @@ export class PlansStep extends Component {
 			planTypes,
 			flowName,
 			showTreatmentPlansReorderTest,
+			// eslint-disable-next-line no-unused-vars
 			isLoadingExperiment,
 			isInVerticalScrollingPlansExperiment,
 			isReskinned,
 		} = this.props;
 
+		const loadingPlanDisplay = (
+			<div className="plans__loading-container">
+				<PulsingDot delay={ 400 } active />
+			</div>
+		);
+
+		const treatmentPlanDisplay = (
+			<TabbedPlans
```

Comment:
There are a few props not being used here. e.g. `isAllPaidPlansShown`, `isInSignup`, `isLaunchPage`, etc. Since `TabbedPlans` is still at the experimental stage, let's remove whatever not necessary to keep it as simple as possible.